.gitattributes

```diff
@@ -0,0 +1,40 @@
+# Set the default line ending style to LF (Unix-style)
+* text eol=lf
+
+# Set the encoding for all text files to UTF-8
+* text=unicode
+
+# Ignore all compiled Python files
+*.pyc binary
+
+# Ignore the local settings file
+blog/settings/local.py export-ignore
+
+# Ignore log files
+log/*.log export-ignore
+
+# Ignore media files
+media/* export-ignore
+
+# Ignore database files
+db.sqlite3 export-ignore
```

Comment:
If these are properly gitignored, there's no need for the export instructions either, I think.

Reply:
Removed